clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -135,6 +143,13 @@ func (*schedulerLatencyMeasurement) String() string {
 }
 
 func (s *schedulerLatencyMeasurement) resetSchedulerMetrics(c clientset.Interface, host, provider, masterName string, masterRegistered bool) error {
+	s.e2eSchedulingDurationInitHist = measurementutil.NewHistogram(nil)
+	s.schedulingAlgorithmDurationInitHist = measurementutil.NewHistogram(nil)
+	s.preemptionEvaluationInitHist = measurementutil.NewHistogram(nil)
+	s.frameworkExtensionPointDurationInitHist = make(map[string]*measurementutil.Histogram)
+	for _, ePoint := range extentionsPoints {
+		s.frameworkExtensionPointDurationInitHist[ePoint] = measurementutil.NewHistogram(nil)
+	}
```

Comment:
do you still need to call DELETE for metrics below/

Reply:
Actually not. But just do not want to break previous version. Will address this.